docs/storefront/catalyst/getting-started/guides/customer-group-component.mdx

```diff
@@ -0,0 +1,105 @@
+- **Simulated group:** This will control which available groups you would like to individually simulate in the editor. The additional selections are included in this control including:
+
+    By default, the **No group** selection is included, which includes target customers that are not logged in or have “No group” selected as their BigCommerce group.
+
+Once that’s finished, let’s add “Group A” to the control for **Selected groups**, and simulate it with the **Simulate group** control. Once simulated, let's add the text “A big discount for group A” that only Group A customers can see.  
+
+
+<Video src={'https://storage.googleapis.com/bigcommerce-production-dev-center/videos/catalyst/customer-group-component/group-A.mp4'}/>
+
+We are now in control of what **Group A** will see.
+
+Repeat the same steps for **Group B**:
+
+<Video src={"https://storage.googleapis.com/bigcommerce-production-dev-center/videos/catalyst/customer-group-component/group-B.mp4"} />
+
+You can now control what users belonging to **Group A** and **Group B** will see, respectively.
+
+Let’s also define the content to be shown for a customer that doesn’t belong to a group by targeting **No group**.  
+
+<Video src={"https://storage.googleapis.com/bigcommerce-production-dev-center/videos/catalyst/customer-group-component/no-group.mp4"} />
+
+## Testing your changes
+
+Publish your changes with the “Publish” button on the top right and make sure you select the page you’ve made changes to.
+
+<Video src={"https://storage.googleapis.com/bigcommerce-production-dev-center/videos/catalyst/customer-group-component/publish.mp4"}/>
+
+To officially test that this component works, we can view what each group sees by logging into Moon’s, Suzy’s, and Peter’s accounts to ensure the component is rendering as expected for each group. Let’s click the button next to the URL to navigate to this page.  
+
+![Makeswift navigation bar menu](https://storage.googleapis.com/bigcommerce-production-dev-center/images/catalyst/customer-group-component/makeswift-navigation-bar-menu.png)
+
+Before logging in, we can see the content that is rendering for customers with no group.
+
+![Rendered content without logging in](https://storage.googleapis.com/bigcommerce-production-dev-center/images/catalyst/customer-group-component/no-group-storefront.png)
+
+Now let’s log into Suzy’s account and see what she will see.
+
+<Video src={"https://storage.googleapis.com/bigcommerce-production-dev-center/videos/catalyst/customer-group-component/group-B-storefront.mp4"} />
+
+Great! How does Moon’s account look?
+
+<Video src={"https://storage.googleapis.com/bigcommerce-production-dev-center/videos/catalyst/customer-group-component/group-A-storefront.mp4"} />
+
+And finally, Peter’s account which doesn’t belong to a group.  
+
+<Video src={"https://storage.googleapis.com/bigcommerce-production-dev-center/videos/catalyst/customer-group-component/no-group-storefront.mp4"} />
```

Comment:
No need to update this, but in the future, let's keep these the same resolution as the rest of the videos 16:9